packages/vaadin-accordion/test/accordion.test.js

```diff
@@ -40,12 +40,18 @@ describe('vaadin-accordion', () => {
   });
 
   describe('custom element definition', () => {
-    it('should define a custom element with proper tag name', () => {
-      expect(customElements.get('vaadin-accordion')).to.be.ok;
+    let tagName;
+
+    beforeEach(() => {
+      tagName = accordion.tagName.toLowerCase();
+    });
+
+    it('should be defined in custom element registry', () => {
+      expect(customElements.get(tagName)).to.be.ok;
```

Comment:
In case the element wasn't properly defined in the registry, this test case won't be executed as `accordion._observer.flush();` throws an `undefined` exception before.

For that reason, I would suppose, the `custom element definition` section should rather create its own very "basic" fixture.

Reply:
Updated the tests to not use `_observer` as this is a private API, now it uses the `nextFrame` instead.